image-layout.md

```diff
@@ -78,13 +78,13 @@ $ cat ./blobs/sha256-afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.manifest.v1+json",
   "config": [
-    "mediaType": "application/vnd.oci.image.serialization.config.v1+json",
+    "mediaType": "application/vnd.oci.image.config.v1+json",
     "size": 7023,
     "digest": "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270"
   },
   "layers": [
     {
-      "mediaType": "application/vnd.oci.image.serialization.rootfs.tar.gzip",
+      "mediaType": "application/vnd.oci.image.layer.tar+gzip",
```

Comment:
It looks like the `+{suffix}` format is from [RFC 6839][1].  That doesn't have any examples of nested formats, but it doesn't specify a formal syntax for suffixes either.  Since the layer idea doesn't seem particularly tied to tar *or* gzip, I'd rather use `application/vnd.oci.image.layer+tar+gzip` to make it easy to also support `+zip` or `+tar+xz`, etc.

[1]: https://tools.ietf.org/html/rfc6839